source/Ship.cpp

```diff
@@ -721,7 +721,7 @@ void Ship::Place(Point position, Point velocity, Angle angle)
 	slowness = 0.;
 	isInvisible = !HasSprite();
 	jettisoned.clear();
-	hyperspaceCount = 0;
+	hyperCount = 0;
```

Comment:
Why rename this variable? `hyperCount` is less clear than `hyperspaceCount`

Reply:
Consistency: A lot of variables needed to be prefix with "hyperspace" to remove ambiguity, but formulas were becoming very long with the extensive variable names, so those variables were instead just prefixed with "hyper". `hyperspaceCount` was the only exception to the prefixing pattern, so to improve code consistency it was changed.

Reply:
Use multiple lines.
General style used here is a double indent relative to the first line.
```
this = other
        + stuff
        + more stuff
```

Reply:
Multiple lines for what?
There's no formulas in this code long enough to justify the usage of multiple lines.

Reply:
You just said they where getting very long, and now you are saying that they are not long enough to use more than one line.